android/src/main/kotlin/com/airship/flutter/Utils.kt

```diff
@@ -0,0 +1,76 @@
+package com.airship.flutter
+
+import android.os.Build
+import android.os.Bundle
+import android.service.notification.StatusBarNotification
+import androidx.annotation.RequiresApi
+import com.urbanairship.util.UAStringUtil
+import com.urbanairship.push.PushMessage
+import org.json.JSONException
+
+class Utils {
+
+    companion object {
+        val shared = Utils()
+    }
+
+    @RequiresApi(Build.VERSION_CODES.KITKAT)
+    fun messageFromNotification(statusBarNotification: StatusBarNotification): PushMessage {
```

Comment:
You should either move these into the companion object to be static, or probably better would be to make an extension for the StatusBarNotfiication to pull out the PushMessage.